src/stdlib/distributions.jl

```diff
@@ -49,3 +49,49 @@ end
 function Distributions.logpdf(d::VecBinomialLogit{<:Real}, ks::Vector{<:Integer})
     return sum(logpdf_binomial_logit.(d.n, d.logitp, ks))
 end
+
+# OrderedLogistic distribution
+struct OrderedLogistic{T1, T2} <: DiscreteUnivariateDistribution
+   η::T1
+   cutpoints::Vector{T2}
+end
+
+# logpdf of the OrderedLogistic distribution
+function Distributions.logpdf(d::OrderedLogistic, k::Int)
+
+    K = length(d.cutpoints)+1
+
+    c =  d.cutpoints
+
+    if k==1
+        logp= log(logistic(c[k]-d.η))
```

Comment:
If I'm not mistaken, these `log(logistic(x))` functions could be replaced with `log1pexp` a.k.a. `softplus` from StatsFuns.jl